indico/modules/events/sessions/util.py

```diff
@@ -153,3 +154,64 @@ def get_sessions_for_user(event, user):
                         ~Session.is_deleted)
                 .all())
     return {sess for sess in sessions if any(user in entry.principal for entry in iter_acl(sess.acl_entries))}
+
+
+def serialize_session_for_ical(sess):
+    session_data = {
+        '_fossil': 'sessionMetadata',
+        'id': sess.id,
+        'startDate': sess.start_dt,
+        'endDate': sess.end_dt,
+        'url': '',
+        'title': sess.title,
+        'location': sess.venue_name,
+        'roomFullname': sess.room_name,
+        'description': ''
+    }
+
+    description = ''
+    for c in sess.contributions:
+        if c.speakers:
+            speakers = ('{} ({})'.format(s.person.full_name, s.person.affiliation) for s in c.speakers)
+            description += 'Speakers: {}\n'.format(', '.join(speakers))
+
+    session_data['description'] = description
+    return session_data
+
+
+def serialize_session(sess):
```

Comment:
I don't think this belongs in here. This output is legacy and only used in the timetable, so I would just put it within `TimetableSerializer`.